src/AmazonPHP/SellingPartner/ObjectSerializer.php

```diff
@@ -342,6 +344,16 @@ public static function deserialize(Configuration $configuration, $data, string $
         }
 
         if (\method_exists($class, 'getAllowableEnumValues')) {
+            $brokenModelDefinitions = [
+                EventCode::class, // https://github.com/amazon-php/sp-api-sdk/issues/191
```

Comment:
@Tetsuya-Takiguchi here is the EventCode class, you are saying that $class variable is getting different value?

Reply:
@norberttech 
Yes, I am. That is correct.
The value of the `$class` variable is `\AmazonPHP\SellingPartner\Model\FulfillmentOutbound\EventCode`, with a backslash at the beginning of the string.
In contrast, the value that can be retrieved from `EventCode::class` is `AmazonPHP\SellingPartner\Model\FulfillmentOutbound\EventCode`, with no backslash at the beginning of the string.

Reply:
oh now I see, thanks! I adjusted the logic to trim left backslash before comparison which should solve the problem. 
https://github.com/amazon-php/sp-api-sdk/pull/194

Reply:
@norberttech 
Thank you for the correction.
I will pull #194 and check it out.